frontend/src/pages/dashboard/jios/form/JiosSearchForm.tsx

```diff
@@ -19,41 +18,42 @@ import { useForm } from 'react-hook-form';
 import { Gym } from '../../../../@types/gym';
 // dayjs
 import { useSnackbar } from 'notistack';
-import { LoadingButton } from '@mui/lab';
 import { useSelector } from '../../../../store';
-import { dateToTimeString, setDateTime } from '../../../../utils/formatTime';
-import { JioSearchFormValues, JIOTYPE_OPTION } from './utils';
-import { addHours } from 'date-fns';
+import { setDateTime } from '../../../../utils/formatTime';
+import { JioSearchFormValues, JIOTYPE_OPTION, yupStartEndDateTimingObject } from './utils';
+import { addDays } from 'date-fns';
+import { IconStyle } from '../../../../sections/@dashboard/user/profile/common';
+import FloatingBottomCard from '../../../../components/FloatingBottomCard';
 
 type Props = {
   onSubmit: (data: JioSearchFormValues) => Promise<void>;
   submitIcon: React.ReactElement;
   submitLabel: string;
+  onClear: () => void;
   defaultValues?: JioSearchFormValues;
 };
 
 export default function JiosSearchForm({
   onSubmit,
   defaultValues: currentJio,
   submitIcon,
+  onClear,
   submitLabel,
 }: Props) {
   const { enqueueSnackbar } = useSnackbar();
   const gyms = useSelector((state) => state.gyms.data);
 
   const NewJioSchema = Yup.object().shape({
```

Comment:
Perhaps we should standardize to use camelCase instead of PascalCase ?

Reply:
Agree.